lib/nexmo_developer/app/views/static/podcast.html.erb

```diff
@@ -1,40 +0,0 @@
-<div class="container">
```

Comment:
I think we need this one as well, we have a controller action for it.

Reply:
It's unlinked and not referenced anywhere, I believe it was used for an event pre-config driven landing pages days.

Reply:
Looks like it was used for `/hansel` See [this commit](https://github.com/Nexmo/nexmo-developer/commit/dd13c32bd92fb2db2e7b62f358437c1d785980c7).

That url is now handled as a landing page. Can you remove the controller action?

Reply:
Yes, good catch. Will do now.

Reply:
Done ✅